### PR TITLE
feature: added swagger documentation for sequelize movement

### PR DESCRIPTION
**Pull request by IOVgomezdn**
added endpoints documentation in sequelize movement's route file

https://rsklabs.atlassian.net/browse/TQAS-45

### CODE REVIEW

**Review by gaston-areco-iov — CHANGES_REQUESTED** [review]
Swagger descriptions should be placed above each corresponding route

**Comment by IOVgomezdn** [comment]
> Swagger descriptions should be placed above each corresponding route

done

**Comment by IOVgomezdn** [comment]
> Swagger descriptions should be placed above each corresponding route

actually done this time!

File: src/routes/sequelize/movement.route.sequelize.ts [hunk 1]
```diff
@@ -8,12 +8,125 @@ import {
...
+ *        description: The movement's ID
+ *    responses:
+ *      200:
+ *        description: Movement found
+ *        content:
+ *          application/json:
+ *            schema:
+ *              type: object
+ *              $ref: '#/components/schemas/Movement'
+ *      400:
```
Comment (gaston-areco-iov) [inline src/routes/sequelize/movement.route.sequelize.ts]: 404 based on controller response code

**Review by gaston-areco-iov — CHANGES_REQUESTED** [review]

**Review by gaston-areco-iov — APPROVED** [review]

**Pull request merged**